docs/docs/vulnerability-scanning/trivy.md

```diff
@@ -89,6 +89,9 @@ EOF
 | `trivy.skipFiles`                     | N/A                                | A comma separated list of file paths for Trivy to skip traversal.                                                                                                                                                                                                                                                                                                                     |
 | `trivy.skipDirs`                      | N/A                                | A comma separated list of directories for Trivy to skip traversal.                                                                                                                                                                                                                                                                                                                    |
 | `trivy.ignoreFile`                    | N/A                                | It specifies the `.trivyignore` file which contains a list of vulnerability IDs to be ignored from vulnerabilities reported by Trivy.                                                                                                                                                                                                                                                 |
+| `trivy.ignorePolicy`                  | N/A                                | It specifies a fallback [policy](https://aquasecurity.github.io/trivy/latest/docs/vulnerability/examples/filter/#by-open-policy-agent) file which allows to customize whicht vulnerabilities are reported by Trivy.                                                                                                                                                                   |
+| `trivy.ignorePolicy.{ns}`             | N/A                                | It specifies a namespace specific [policy](https://aquasecurity.github.io/trivy/latest/docs/vulnerability/examples/filter/#by-open-policy-agent) file which allows to customize whicht vulnerabilities are reported by Trivy.                                                                                                                                                         |
+| `trivy.ignorePolicy.{ns}.{wl}`        | N/A                                | It specifies a namespace/workload specific [policy](https://aquasecurity.github.io/trivy/latest/docs/vulnerability/examples/filter/#by-open-policy-agent) file which allows to customize whicht vulnerabilities are reported by Trivy.                                                                                                                                                |
```

Comment:
typo `whicht` -- > which

also could you please update[ helm docs](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/README.md) (for artifacthub)

Reply:
@chen-keinan Helm docs updated as well as the templates